heidrun/uva_mods.rb

```diff
@@ -0,0 +1,180 @@
+# coding: utf-8
+
+#   <accessCondition type="restrictionOnAccess"
+#                    displayLabel="Access to the Collection"/>;
+#   <accessCondition type="useAndReproduction"
+#                    displayLabel="Use of the Collection">
+rights_map = lambda do |record|
+  rights = record['mods:accessCondition']
+           .match_attribute(:type, 'restrictionOnAccess')
+           .match_attribute(:displayLabel, 'Access to the Collection')
+
+  rights.concat(record['mods:accessCondition']
```

Comment:
That's right. This is a proxy for Ruby's `Array#concat`. See: https://github.com/dpla/KriKri/blob/develop/lib/krikri/parser.rb#L201-L206